wis2box_api/plugins/process/bufr2geojson.py

```diff
@@ -23,6 +23,7 @@
 import requests
 import base64
 
+from wis2box_api.wis2box.env import STORAGE_PUBLIC_URL
```

Comment:
Imports should be in the order of (alphabetical, with blank line between groupings):

- standard library
- 3rd party packages/modules
- local packages/modules

So:

```python
import base64
import requests

from bufr2geojson import transform as as_geojson
from pygeoapi.process.base import BaseProcessor

from wis2box_api.wis2box.env import STORAGE_PUBLIC_URL
```